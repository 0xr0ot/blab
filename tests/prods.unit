S = NP s VP
NP = N 
N = A s N2
A = "a"
N2 = "cat"
V = "saw"
VP = V s N
s = 32
