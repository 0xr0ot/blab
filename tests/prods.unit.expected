a cat saw a cat